app/src/main/kotlin/com/simplemobiletools/musicplayer/activities/TrackActivity.kt

```diff
@@ -293,22 +297,30 @@ class TrackActivity : SimpleActivity(), PlaybackSpeedListener {
             applyColorFilter(if (isShuffleEnabled) getAdjustedPrimaryColor() else config.textColor)
             alpha = if (isShuffleEnabled) 1f else MEDIUM_ALPHA
             contentDescription = getString(if (isShuffleEnabled) R.string.disable_shuffle else R.string.enable_shuffle)
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
can be replaced by "isOreoPlus()" from Simple Commons

Reply:
Good to know, thanks. But won't be needed since we're removing the tooltips anyway